sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/CountAggregator.java

```diff
@@ -8,54 +8,62 @@
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
 import io.opentelemetry.sdk.metrics.data.AggregationTemporality;
+import io.opentelemetry.sdk.metrics.data.Exemplar;
 import io.opentelemetry.sdk.metrics.data.LongSumData;
 import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.exemplar.ExemplarReservoir;
 import io.opentelemetry.sdk.metrics.internal.descriptor.MetricDescriptor;
 import io.opentelemetry.sdk.resources.Resource;
+import java.util.List;
 import java.util.Map;
 import java.util.concurrent.atomic.LongAdder;
+import java.util.function.Supplier;
 import javax.annotation.concurrent.ThreadSafe;
 
 @ThreadSafe
-final class CountAggregator extends AbstractAggregator<Long> {
+final class CountAggregator extends AbstractAggregator<LongAccumulation> {
   private final AggregationTemporality temporality;
+  // Workaround
+  private final Supplier<ExemplarReservoir> reservoirBuilder;
 
   CountAggregator(
       Resource resource,
       InstrumentationLibraryInfo instrumentationLibraryInfo,
       MetricDescriptor descriptor,
-      AggregationTemporality temporality) {
+      AggregationTemporality temporality,
+      Supplier<ExemplarReservoir> reservoirBuilder) {
     super(
         resource,
         instrumentationLibraryInfo,
         descriptor,
         temporality == AggregationTemporality.CUMULATIVE);
     this.temporality = temporality;
+    this.reservoirBuilder = reservoirBuilder;
   }
 
   @Override
-  public AggregatorHandle<Long> createHandle() {
-    return new Handle();
+  public AggregatorHandle<LongAccumulation> createHandle() {
```

Comment:
Are we keeping the door open to not require the `LongAccumulation` wrapper when exemplars are turned off for a given instrument?

Reply:
(this question applies to all the aggregators of course..I just thought about it when looking at this one).

Reply:
Good question.   I wasn't planning to. I need to re-run the benchmark, as we're effectively boxing over the Long here anyway so I didn't see a huge slowdown.   This accumulations is NOT on the critical path, only when collecting.